components/admin_console/permission_schemes_settings/permission_system_scheme_settings/permission_system_scheme_settings.jsx

```diff
@@ -331,7 +332,7 @@ export default class PermissionSystemSchemeSettings extends React.PureComponent
                             </div>
                         </div>
 
-                        {this.props.license && this.props.config.EnableGuestAccounts === 'true' &&
+                        {isLicensed && this.props.config.EnableGuestAccounts === 'true' &&
```

Comment:
Not familiar with this codebase but wanted to confirm the intention here is to keep this under the license check

Reply:
Yes, the guests permissions widget on this page should only be visible for licensed instances. It looks like there was a pre-existing but that would show the widget if the `license` prop was present, even if `IsLicensed` was `false`. I guess it wasn't discovered because the entire page used to be behind a license check.